modules/core/src/parallel.cpp

```diff
@@ -267,7 +301,7 @@ static SchedPtr pplScheduler;
 
 void cv::parallel_for_(const cv::Range& range, const cv::ParallelLoopBody& body, double nstripes)
 {
-    CV_INSTRUMENT_REGION()
+    CV_INSTRUMENT_REGION_MT_FORK()
```

Comment:
Could you add this code:
```
if (range.empty())
    return;
```

It is called with zero range from [here](https://github.com/opencv/opencv/blob/d1bbc0b6e6c0c5115e54aa6d9f32a1d332a9538c/modules/features2d/src/kaze/KAZEFeatures.cpp#L567) and crashed on line 207 (div by zero).